src/main/kotlin/com/cjbooms/fabrikt/generators/OasDefault.kt

```diff
@@ -39,7 +39,7 @@ sealed class OasDefault {
 
     data class EnumValue(val type: TypeName, val enumValue: String) : OasDefault() {
         override fun getDefault(): CodeBlock =
-            CodeBlock.of("%T.${enumValue.toUpperCase()}", type)
+            CodeBlock.of("%T.${enumValue.toEnumName()}", type)
```

Comment:
When referencing the enum item, the name should be formed the same way as when that item is generated – using `toEnumName()` method.